packages/app-project/src/components/AuthModal/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,122 @@
+import { SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import { Formik } from 'formik'
+import { Anchor, Box, Button, Heading, Text, FormField, TextInput } from 'grommet'
+import Link from 'next/link'
+import { func } from 'prop-types'
+import React from 'react'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+const userNameFieldId = 'LoginForm_login'
+const passwordFieldId =  'LoginForm_password'
+
+const textColors = { dark: 'white', light: 'black' }
+
+const Label = (props) => (
+  <SpacedText
+    color={textColors}
+    weight='bold'
+    {...props}
+  />
+)
+
+function LoginForm (props) {
+  const { onSubmit } = props
+  return (
+    <Box>
+      <Heading size='small' color={textColors} margin={{ bottom: 'xsmall', top: 'none' }}>
+        Welcome back!
+      </Heading>
+      <Text color={textColors}>
+        We're glad you're back! Sign in to pick up where you left off.
+      </Text>
+
+      <Formik
+        initialValues={{ login: '', password: '' }}
+        validate={validateLogin}
+        onSubmit={onSubmit}
+      >
+        {({
+          errors,
+          handleChange,
+          handleBlur,
+          handleSubmit,
+          isSubmitting,
+          touched,
+          values,
+          /* and other goodies */
+        }) => (
+          <Box as='form' onSubmit={handleSubmit} margin={{ top: 'small' }}>
+
+            <FormField
+              error={errors.login && touched.login && errors.login}
+              htmlFor={userNameFieldId}
+              label={<Label>Username or email address</Label>}
+            >
+              <TextInput
+                disabled={isSubmitting}
```

Comment:
We have a regression here on the focus functionality... Just tried `autoFocus` again here, and this time it worked, so I'm not 100% why it was breaking in my branch and PR.

Reply:
Fixed via #888